astropy/units/typing_utils.py

```diff
@@ -76,9 +69,15 @@ def generate_stub_lines(module) -> Iterator[str]:
             # the class of the unit object (e.g., IrreducibleUnit) is often
             # not defined in the module itself, so we need to import it
             import_lines.add(make_import_line(cls))
+
+        elif name == "enable":
+            variable_lines.append("enable() -> _UnitContext: ...")
```

Comment:
Because of this addition `astropy/units/imperial.pyi` contains invalid syntax, so it is not even a valid stub file anymore. I don't know if you tested the stubs or not. If you did and the tests approved them despite invalid syntax then the tests are worthless. I suggest you don't try to make any changes to the contents of the generated stubs in this pull request.

Reply:
OK, I removed it; I'll leave it for typing afficionados to add proper checking (and then add `enable`).